lib/src/values/logic_value.dart

```diff
@@ -268,12 +268,27 @@ abstract class LogicValue {
   }
 
   /// Returns the `i`th bit of this [LogicValue]
+  ///
+  /// The [index] provided can be positive or negative. For positive [index], the
+  /// indexing is performed from front of the LogicValue. For negative [index],
+  /// the indexing started from last index and goes to front.
+  /// Note: the [index] value must follow, -width <= [index] < width
+  ///
+  /// ```dart
+  /// LogicValue.ofString('1111')[2];  // equals(LogicValue.one)
```

Comment:
The `equals` syntax is from the `test` package, which is not really conventional for an example in a doc comment.  Something like `== LogicValue.one` might make more sense.

Reply:
will change it